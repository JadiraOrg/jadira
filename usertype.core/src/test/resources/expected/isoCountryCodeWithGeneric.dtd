<!ELEMENT dataset (
    ISOCOUNTRYCODEWITHGENERIC*)>

<!ELEMENT ISOCOUNTRYCODEWITHGENERIC EMPTY>
<!ATTLIST ISOCOUNTRYCODEWITHGENERIC
    ID CDATA #REQUIRED
    ISOCOUNTRYCODE CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

