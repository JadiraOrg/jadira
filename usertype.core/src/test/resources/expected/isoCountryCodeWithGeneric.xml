<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "isoCountryCodeWithGeneric.dtd">
<dataset>
  <isoCountryCodeWithGeneric ID="0" ISOCOUNTRYCODE="AD" NAME="test_0"/>
  <isoCountryCodeWithGeneric ID="1" ISOCOUNTRYCODE="GB" NAME="test_1"/>
  <isoCountryCodeWithGeneric ID="2" ISOCOUNTRYCODE="US" NAME="test_2"/>
  <isoCountryCodeWithGeneric ID="3" ISOCOUNTRYCODE="FR" NAME="test_3"/>
  <isoCountryCodeWithGeneric ID="4" NAME="test_4"/>
</dataset>